docker/pdo_services_base.dockerfile

```diff
@@ -59,8 +59,7 @@ ENV SGX_SDK=/opt/intel/sgxsdk
 #   has 2.30 but Intel ships binary distro for 2.32.51.20190719
 # -----------------------------------------------------------------
 WORKDIR /opt/intel
-RUN [ "$UBUNTU_VERSION" = "20.04" ] \
-  && SGX_SDK_BINUTILS_REPO=https://download.01.org/intel-sgx/sgx-linux/${SGX} \
+RUN SGX_SDK_BINUTILS_REPO=https://download.01.org/intel-sgx/sgx-linux/${SGX} \
```

Comment:
you're just dropping the test here, right? tbh, i'm not sure why we make the ubuntu version a build arg. i suppose it doesn't hurt & i've used it for testing 22.04 support for the client. but... i doubt very much that anything newer that 22.04 would work out of the box.

Reply:
> you're just dropping the test here, right? tbh, i'm not sure why we make the ubuntu version a build arg. i suppose it doesn't hurt & i've used it for testing 22.04 support for the client. but... i doubt very much that anything newer that 22.04 would work out of the box.

This test afaik is a legacy of 18.04 or even 16.04 where packaging was different and there was nothing really 20.04 specific (except the time this was originally added it was the latest version)

Reply:
Perfect.